scout/server/blueprints/clinvar/templates/clinvar_submissions.html

```diff
@@ -77,187 +81,199 @@ <h5>Manual submission</h5>
 {% endmacro %}
 
 {% macro submission_panel(subm_obj) %}
-  <div class="card w-100 mt-3">
```

Comment:
This looks like a lot of changes but it isn't. It's exactly the same stuff but inside an accordion, to give a less messy layout:

<img width="1225" alt="image" src="https://user-images.githubusercontent.com/28093618/221807835-46633e9f-fbc9-42c5-8a43-65b6869f2b1e.png">